draft-ietf-gnap-resource-servers.md

```diff
@@ -236,19 +236,40 @@ Cache-Control: no-store
 }
 ~~~
 
-## Registering a Resource Handle {#rs-register-resource-handle}
+## Registering a Resource Set {#rs-register-resource-handle}
```

Comment:
Do we think resource registration will be available on all Authz Servers? Should we make it an explicitly optional capability? How?

Reply:
Everything in the RS draft is an optional capability for the AS already. It could be indicated by the presence/absence of URLs in the discovery document.

Reply:
Last I checked some of these URLs were explicitly marked as Optional, and some weren't.